CHANGELOG: Add an entry for the redesign (#22003)
sourcegraph/sourcegraph: CHANGELOG.md

- Pings removes the collection of aggregate search filter usage counts and adds a smaller set of aggregate usage counts for query operators, predicates, and pattern counts. [#21320](https://github.com/sourcegraph/sourcegraph/pull/21320)
- Sourcegraph will now refuse to start if there are unfinished [out-of-band-migrations](https://docs.sourcegraph.com/admin/migrations) that are deprecated in the current version. See the [upgrade documentation](https://docs.sourcegraph.com/admin/updates) for changes to the upgrade process. [#20967](https://github.com/sourcegraph/sourcegraph/pull/20967)
- Code Insight pages now have new URLs [#21856](https://github.com/sourcegraph/sourcegraph/pull/21856)
- We are proud to bring you [an entirely new visual design for the Sourcegraph UI](https://about.sourcegraph.com/blog/introducing-sourcegraphs-new-ui/). We think you’ll find this new design improves your experience and sets the stage for some incredible features to come. Some of the highlights include:

>   - **Refined search results:** The redesigned search bar provides more space for expressive queries, and the new results sidebar help users discover search syntax without referencing documentation.

Reviewer: ```suggestion
  - **Refined search results:** The redesigned search bar provides more space for expressive queries, and the new results sidebar helps to discover search syntax without referencing documentation.
```

  - **Improved focus on code:** We’ve reduced non-essential UI elements to provide greater focus on the code itself, and positioned the most important items so they’re unobtrusive and live exactly where they are needed.

>   - **Improved layouts:** We’ve improved pages like diff diff views to make them easier to use and to help users quickly find the information they need.

Reviewer: ```suggestion
  - **Improved layouts:** We’ve improved pages like diff views to make them easier to use and to help find information quickly.
```

- We are proud to bring you [an entirely new visual design for the Sourcegraph UI](https://about.sourcegraph.com/blog/introducing-sourcegraphs-new-ui/). We think you’ll find this new design improves your experience and sets the stage for some incredible features to come. Some of the highlights include:

  - **Refined search results:** The redesigned search bar provides more space for expressive queries, and the new results sidebar helps to discover search syntax without referencing documentation.
  - **Improved focus on code:** We’ve reduced non-essential UI elements to provide greater focus on the code itself, and positioned the most important items so they’re unobtrusive and live exactly where they are needed.
  - **Improved layouts:** We’ve improved pages like diff views to make them easier to use and to help find information quickly.
  - **New navigation:** A new global navigation provides immediate discoverability and access to current and future functionality.
  - **Promoting extensibility:** We've brought the extension registry back to the main navigation and improved its design and navigation.

>   With bulk of the redesign complete, future releases will include more improvements along these lines.

Reviewer: ```suggestion
  With bulk of the redesign complete, future releases will include more improvements and refinements.
```
